dns/bind/src/opnsense/scripts/OPNsense/Bind/dnsbl.sh

```diff
@@ -88,13 +88,22 @@ install() {
 
 for CAT in $(echo ${1} | tr ',' ' '); do
 	case "${CAT}" in
-	ad)
+	ag)
+	        adguard
+		;;
+	el)
 		easylist
+		;;
+        ep)
 		easyprivacy
+		;;
+        nc)
```

Comment:
the previous two seems to have an additional white space in the compare value

Reply:
dont see it???

Reply:
https://github.com/opnsense/plugins/commit/520a51be1